config/manifests/bases/group-sync-operator.clusterserviceversion.yaml

```diff
@@ -809,6 +816,53 @@ spec:
     mediatype: image/png
   install:
     spec:
+      clusterPermissions:
```

Comment:
this is almost certainly wrong, permissions will be generated on the fly based on annotations.

Reply:
I believe this is related to the error that you had passed along previously